README.rst

```diff
@@ -24,11 +24,12 @@ file of your project:
     {
         public function registerBundles()
         {
-            $bundles = array(
-                // ...
+            //...
+            if (in_array($this->getEnvironment(), ['dev', 'test'])) {
+                //...
+                $bundles[] = new Hautelook\AliceBundle\HautelookAliceBundle();
```

Comment:
Because the vendor code could be not present, you should check the bundle is here before using it.